wagtail/wagtailadmin/templates/wagtailadmin/shared/search_area.html

```diff
@@ -0,0 +1,6 @@
+{% if not active %}
```

Comment:
Now that the 'Other searches' bar sticks around, having the search types appear and disappear as you click through them makes checking through all the search types quite difficult. As 'Document' is much longer than 'Page', switching from one to the other makes all subsequent search types shift on screen dramatically. Could the search type bar be changed to a 'Category', 'Search for:', or other wording; and the current search type stay in the list, but bolded/not linked to show it was the current search type?